### DISCUSSION TITLE
Icarus: does it work with the Jakarta version

### DISCUSSION THREAD

**Discussion by cloudsistem** [primefaces/community#1351] [PrimeFaces Templates] [answered]
I would like to know if it's possible to use the Icarus theme with the updated Jakarta version of Java.

**Comment by melloware** [comment, marked as answer]
Yes download the latest version of the theme from the PrimeStore and it comes with a `-jakarta.jar` version of the Theme JAR file for using with Jakarta EE10+

**Comment by souflaz** [comment]
in primestore , all i find is the icarus 2.0.1 version with no jakarta version

**Reply by melloware** [reply]
I assume this means Icarus is a very old retired template?

**Reply by souflaz** [reply]
Exactly, it is one of the retired templates

**Reply by melloware** [reply]
OK i worked with @souflaz to upgrade it its a VERY old template but its still possible to migrate to PF15.0.0 it just takes some more work.  

I recommend using this template to get up to PF 15.0.0 but then you should look at migrating to a more modern template.
